dev-python/numba/numba-0.49.1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# enable 3.8 when llvmlite gets 3.8.
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="NumPy aware dynamic Python compiler using LLVM"
+HOMEPAGE="http://numba.pydata.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="examples openmp threads"
+
+distutils_enable_tests pytest
+
+RDEPEND="
+	>=dev-python/llvmlite-0.32.0[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	threads? ( dev-cpp/tbb )
+	openmp? ( virtual/mpi[cxx] )
+"
+BDEPEND="
+	${RDEPEND}
+	test? (
+		sci-libs/scipy[${PYTHON_USEDEP}]
+	)
+"
+
+PATCHES=(
+	"${FILESDIR}/numba-skip-tests.patch"
+	"${FILESDIR}/numba-tbb-check.patch"
```

Comment:
Your patch should always have a version where it got introduced, and preferably headers still intact if it was taken from upstream.

Reply:
its not from upstream, they haven't responded to my request yet.

Reply:
the skip tests patch is an old one, so I did not touch that. I've added the version for the other one.